src/StackExchange.Redis/Enums/ReplicationChangeOptions.cs

```diff
@@ -21,12 +21,17 @@ public enum ReplicationChangeOptions
         /// </summary>
         Broadcast = 2,
         /// <summary>
-        /// Issue a SLAVEOF to all other known nodes, making this this master of all
+        /// Issue a REPLICAOF to all other known nodes, making this this master of all
         /// </summary>
+        [Obsolete(Messages.PreferReplica)]
         EnslaveSubordinates = 4,
         /// <summary>
+        /// Issue a REPLICAOF to all other known nodes, making this this master of all
+        /// </summary>
+        ReplicateToSubordinates = 4, // note ToString prefers *later* options
```

Comment:
`FormReplicaChain`? I don't know that I have an actual better suggestion here - whatever you think